건우/Week2_중복된숫자개수.js

```diff
@@ -0,0 +1,9 @@
+function solution(array, n) {
+    var answer = 0;
+    for (i=0; i<array.length; i++) {
+        if (array[i] == n) {
+            answer = answer +1;
+        }
+    }
```

Comment:
for문을 사용하지 못하는건 아니지만, 해당 문제에서는 제가 전통적인 for문에 대해서는 사용을 자제하고
배열 내장함수를 추천드렸던 것 같은데요!

해당 내용은
```suggestion
    array.forEach((value) => {
        if (value === n) answer += 1;
    }
```
로도 작성할 수 있습니다.